migrate_pim/changelog.rst

```diff
@@ -0,0 +1,20 @@
+Where is the Changelog?
+========================
+
+When our maintenance team works on issues, they take extreme care to avoid any backward compatibility break, but it might happen. If it is the case, the BC break is mentioned in the changelog of the version.
+
+.. note::
+
+    Akeneo provides 2 changelogs: one for CE edition available the Community repo and one for EE edition which is only available in the latest archive.
```

Comment:
in the latest Enterprise Edition archive.

Reply:
:+1: